mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -736,7 +739,10 @@ def MIOpen_BlockwiseGemmV2Op:
     not slices of a larger object.
   }];
   let assemblyFormat = [{
-    `(` operands `)` attr-dict `:` type(operands) `->` type($vectorDs)
+    $matrixC `+` `` `=` $bufferA `by` $matrixA `[` $waveOffsetA `]` `*`
```

Comment:
Nit: is `by` the right word here? Maybe `from` or `<-`

Reply:
Yes, `from` is better, I used `by` just so that it is as short as possible.

Reply:
I'd still go for `from` or something like `<-` on the theory that it'll make reading the IR clearer, which is rather important.

Reply:
Addressed in a7f8be7